### PR TITLE
Mapping capture background path from calibration to reconstruction

### PR DESCRIPTION
**Pull request by edyoshikun**
Added Signal/Slot to update the acquired background path on the reconstruction from the background capture button in acquisition.

### CODE REVIEW

**Comment by edyoshikun** [comment]
This was using Falcon, for which we updated mmc version 09012022 and the respective components to work with RecOrder.

File: recOrder/plugin/widget/main_widget.py [hunk 1]
```diff
@@ -2112,6 +2112,17 @@ def enter_acq_bg_path(self):
         else:
             self.ui.le_bg_path.setText('Path Does Not Exist')
 
+    @Slot(str)
+    def handle_bg_path_update(self,value):
```
Comment (ziw-liu) [inline recOrder/plugin/widget/main_widget.py:2116]: Consider PEP 8 issue: `(self,value)` -> `(self, value)`.

Comment (ziw-liu) [reply recOrder/plugin/widget/main_widget.py:2116]: Also please include docstrings for your public methods.

File: recOrder/plugin/widget/main_widget.py [hunk 2]
```diff
@@ -2112,6 +2112,17 @@ def enter_acq_bg_path(self):
...
+        #Path from Background Worker
+        path = value
+        if os.path.exists(path):
+            self.acq_bg_directory = path
+            self.current_bg_path = path
+            self.ui.le_bg_path.setText(path)
+        else:
+            self.ui.le_bg_path.setText('Path Does Not Exist')
```
Comment (ziw-liu) [inline recOrder/plugin/widget/main_widget.py:2124]: Refer to: https://github.com/mehta-lab/recOrder/blob/f4d6407ca7d226ffddf04bfc77afc31d677c3aa5/recOrder/plugin/widget/main_widget.py#L2276-L2277

Comment (ziw-liu) [reply recOrder/plugin/widget/main_widget.py:2124]: Ideally we also want a napari-level warning banner for all these events. Maybe worth a separate issue for 1.0.0+.

Comment (ziw-liu) [reply recOrder/plugin/widget/main_widget.py:2124]: On second thought, do we need this if the parameter is only passed in from within `recOrder`? How about a try/except instead?

File: recOrder/plugin/widget/main_widget.py (hunk 1 above)
Comment (edyoshikun) [reply recOrder/plugin/widget/main_widget.py:2116]: Thank you! I'll be mindful for future PRs.

**Review by ziw-liu — APPROVED** [review]
Looks good to me -- great first PR!

**Comment by talonchandler** [comment]
Just tested on hummingbird---works extremely well. 

Great first PR, Ed!

**Review by talonchandler — APPROVED** [review]

**Pull request merged**